python/mlcroissant/mlcroissant/_src/operation_graph/operations/read_test.py

```diff
@@ -21,6 +24,16 @@
 from mlcroissant._src.tests.nodes import empty_file_object
 from mlcroissant._src.tests.operations import operations
 
+CONTENT = """@relation foo
```

Comment:
nit: ARFF_CONTENT and say that the example is from https://docs.scipy.org/doc/scipy-1.13.0/reference/generated/scipy.io.arff.loadarff.html for future editors.

Reply:
Good point, done.